python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -114,7 +93,7 @@ def __copy__(self, memo=None):
     def __deepcopy__(self, memo=None):
         cls = self.__class__
         obj = cls()
-        this_dict = self.__getstate__()
+        this_dict = self.__dict__
```

Comment:
I think this may be the source of the bug of breaking the _parent knowledge in the past as it modifies the object it shouldn't. Either way I'll gladly change